pkg/loghttp/entry.go

```diff
@@ -143,8 +177,15 @@ func (EntryEncoder) Encode(ptr unsafe.Pointer, stream *jsoniter.Stream) {
 	stream.WriteRaw(`"`)
 	stream.WriteMore()
 	stream.WriteStringWithHTMLEscaped(e.Line)
-	stream.WriteMore()
-	stream.WriteString(e.Labels)
+	if len(e.Labels) > 0 {
+		stream.WriteMore()
+		stream.WriteObjectStart()
+		for lName, lValue := range e.Labels {
+			stream.WriteObjectField(lName)
+			stream.WriteString(lValue)
```

Comment:
we need to add `stream.WriteMore()` after this line to add coma between label<>values. except for the last label<>value.

Reply:
Thank you! Fixed and tested with 0b0af55dc0eacf86e583416e5e802d5b883d2f99

Reply:
thanks